presentations/README.md

```diff
@@ -0,0 +1 @@
+# We gather new contributor workshop resources and links on this folder. Patches and better structuring of materials welcome!
```

Comment:
This sentence looks more like a description than a title.

Can we put all content in this single `README.md` file if they're not too much?